src/config/env.ts

```diff
@@ -26,15 +27,16 @@ const requiredEnvVars = [
 	"CRYPTOR_SIDECAR_CLIENT_IDENTIFICATION_CHALLENGE"
 ];
 
-const filename = isNodeTest() ? ".env.test" : ".env";
-const env = dotenv.config({
-	path: path.resolve(process.cwd(), filename)
-});
-
-// TODO: add checks for environment variables here and error out if missing any
-if (env.error && nodeEnv !== EnvironmentEnum.production) {
-	throw env.error;
-}
+// Load environment files
+const envFile = ".env";
+[
+	`${envFile}.${nodeEnv}.local`,
+	`${envFile}.local`,
+	`${envFile}.${nodeEnv}`,
+	envFile
```

Comment:
we do sadly.  We need the .env for the env vars for docker-compose which we use for ngrok_token.  This could be improved upon later with a wrapper or our own ngrok docker service